index.rst

```diff
@@ -21,7 +21,7 @@ NOTE: There are multiple options for reading this documentation. See link to the
    pages/configuration
    pages/secure/securing
    pages/sending_data
-   pages/collector_sidecar
```

Comment:
we should include a `collector_sidecar` page with the `:orphan:` header that gives notice that this is no longer present and a link to the new page.